docs/UserGuide.md

```diff
@@ -45,70 +51,83 @@ Commands in these highlighted boxes are typically used to show what you should e
 `command`<hr>
 
 ## 2.2 TutorAid visual components
+
 This section details the various components in TutorAid and how they will be referred to throughout the guide.
 ![Labelled Ui](images/labelled-ui.jpg)
 
 #### Menu Bar
+
 This is an area where some features can be found. As TutorAid primarily interacts with you through the command box, this area is infrequently used.
 
 #### Command Box
+
 The Command Box is a field in which you can type instructions (commands) to TutorAid. Text in this user guide that is `highlighted` should typically be copied into the command box exactly (including spaces). After you have typed a command into this box, you can press ENTER :leftwards_arrow_with_hook: on your keyboard to tell TutorAid to execute your command.
 
-If your command is of the wrong [format](#23-command-format), TutorAid will try to show you the correct format using the Console if it is able to guess what command you had intended to provide.
+If your command is of the wrong [format](#2.3-command-format), TutorAid will try to show you the correct format in the Console if it is able to infer what command you had intended to provide.
```

Comment:
The 23 wasn't a typo - the hotlink omits the dot. Can you confirm this link still works?

Reply:
Right, I read up a bit on it and realised it is better to leave it as `23` as well, thank you for highlighting!